src/pages/hosts/index.tsx

```diff
@@ -203,6 +232,22 @@ const getQueryVariables = (search: string): HostsQueryVariables => {
   };
 };
 
+// Convert query param values into react-table's column filters state
+const getFilters = (queryParams: HostsQueryVariables): ColumnFiltersState => {
```

Comment:
nit. can you use jsdoc syntax so we get intellisense suggestions? :) 
![image](https://github.com/evergreen-ci/spruce/assets/4605522/701b41e7-4a86-4a2d-913c-e3c3616b34d5)